.gitignore

```diff
@@ -35,6 +35,7 @@ Makefile
 /config.status
 /libtool
 /stamp-h1
+/apps/blueman-*
```

Comment:
This includes all files beginning with blueman-, also the ``.in`` files which I think is not intended here. ``/apps/blueman-*.py`` I think should only include the generated files but exclude the ``'in`` files.